src/ipmanager/ui/templates/ui/single_group_view.html

```diff
@@ -43,11 +44,20 @@ <h2>IP Ranges</h2>
     {% for iprange in ip_ranges %}
       <tr>
         <td>{{iprange.value}}</td>
+        <td>
+          <form method="post" action="{% url 'delete_ip_range' group.key iprange.pk %}"> {% csrf_token %}
+            <button type="delete">Delete</button>
+          </form> 
+        </td>
       </tr>
     {% endfor %}
   </tbody>
 </table>
 </div>
+<form method="post" action="{% url 'create_new_ip_range' group.key %}"> {% csrf_token %}
+  {{ ip_range_form }}
+  <button type="submit">Add IP Range</button>
```

Comment:
This button and the parallel one on the `add_ip_range_form.html` should have the same text and classes:

```
<button type="submit" class="create">Add IP Range</button>
```